noir/noir-repo/compiler/noirc_evaluator/src/brillig/brillig_ir/codegen_control_flow.rs

```diff
@@ -144,25 +146,62 @@ impl BrilligContext {
     pub(crate) fn codegen_constrain_with_revert_data(
         &mut self,
         condition: SingleAddrVariable,
-        assert_message: Option<String>,
+        revert_data_items: Vec<BrilligVariable>,
+        revert_data_types: Vec<BrilligParameter>,
+        revert_data_type: u64,
```

Comment:
This is the selector, correct? We should probably standardise on using that terminology.

Reply:
Yup!